conda/install.py

```diff
@@ -575,7 +575,8 @@ def read_no_link(info_dir):
 # Should this be an API function?
 def symlink_conda(prefix, root_dir, shell=None):
     # do not symlink root env - this clobbers activate incorrectly.
-    if normpath(prefix) == normpath(sys.prefix):
+    # prefix should always be longer than, or outside the root dir.
+    if normpath(prefix) in normpath(root_dir):
```

Comment:
Thanks, that's a good suggestion. I'll get it in for the next bug fix release  -  or submit a pr yourself so you get the credit you should.